client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -502,6 +503,13 @@ export class DomainDetailsForm extends PureComponent {
 		}
 	}
 
+	// We want to cache the functions to avoid triggering unecessary rerenders
+	getInputRefCallback( name ) {
+		return this.inputRefCallbacks[ name ] ||
+			( this.inputRefCallbacks[ name ] = ( el ) =>
+				this.inputRefs[ name ] = el );
```

Comment:
Minor personal preference-ish thing -- can we simplify for readability? Eg, I haven't tested yet, but is this equivalent?

```
if ( this.inputRefCallbacks[ name ] ) {
  return this.inputRefCallbacks[ name ];
}

this.inputRefCallbacks[ name ] = ( el ) => this.inputRefs[ name ] = el;
return this.inputRefCallbacks[ name ];
```

A little longer but worth it so nubs like me can read more easily methinks.